corehq/apps/users/models_role.py

```diff
@@ -81,6 +81,7 @@ class UserRole(models.Model):
     is_archived = models.BooleanField(null=False, default=False)
     upstream_id = models.CharField(max_length=32, null=True)
     couch_id = models.CharField(max_length=126, null=True, default=_uuid_str)
+    is_commcare_user_default = models.BooleanField(null=False, default=False)
```

Comment:
Setting this to `null=False` will result in a query that locks the table while the default is applied. In general it's better to make new columns nullable.